src/Components/CardboardModal/CardboardModal.types.ts

```diff
@@ -24,7 +24,7 @@ export interface ICardboardModalProps {
         linkProps?: ILinkProps;
     };
     /** click handle for the primary footer button */
-    footerPrimaryButtonProps: IButtonProps;
+    footerPrimaryButtonProps?: IButtonProps;
```

Comment:
Should it be possible to have a Modal without buttons? Maybe that is what we should have, in case of having buttons always have primary and secondary, and in case we don't have a primary don't show either.

Reply:
I don't think that we should always have the primary button but disable it if not needed. Since this is a "modal" component, it is possible that we do not need any button but a close button on the top right corner which is the common practice for fluent Modal component. In places where we use it as a "dialog" component, than having buttons (primary & secondary) make sense. In my case, I want to use it as a pure "modal" where only close button is needed with no primary button action. I am in favor of having the flexibility in our custom CardboardModal component which can be either used as a pure modal or dialog by customizing the props. What do you think?

Reply:
I think there should always be a primary action on the screen though right? Even if that action is to close the modal? Maybe we should make it possible to hide the cancel button instead? I can't think of a scenario we have in our app today where we would not want a primary button. What is the scenario you're thinking of here?

Reply:
not necessarily since this is a pure modal which does not require a button at all: https://developer.microsoft.com/en-us/fluentui#/controls/web/modal

The scenario here is that, the "DataHistoryExplorerModal" is going to be triggered from property inspector to visualize some twin data history which does not require a primary action at all, just the close button (or only close button on the top right). This component is self contained in terms of interaction and no need to pass a callback or something. Only requires to be opened and dismiss which is what a modal does (this is not intended to be a dialog type interaction).